hbase-shell/src/main/ruby/hbase/admin.rb

```diff
@@ -1359,26 +1366,26 @@ def list_locks
       @admin.getLocks
     end
 
-    # Parse arguments and update HTableDescriptor accordingly
-    def update_htd_from_arg(htd, arg)
```

Comment:
this method was accessible to shell users, so we will need to release note the removal.

Reply:
Thanks for the heads up! I just update the "Release Note" field in Jira to state that this method was removed.